couchdb-setup.sh

```diff
@@ -58,6 +58,7 @@ curl -X PUT $COUCHURL/resources
 curl -X PUT $COUCHURL/courses
 curl -X PUT $COUCHURL/nations
 curl -X PUT $COUCHURL/communityregistrationrequests
+curl -X PUT $COUCHURL/feedback
```

Comment:
Last time we talked about the naming convention of what we need to do with dcoument name. So could you please follow the style guide. @paulbert might updated it. I think it should be in plural.

Reply:
@nirojdyola This is a tricky part of the English language, but there isn't a different plural for feedback (basically feedback is already plural).  `feedback` is OK for the database name.